docs/conf.py

```diff
@@ -121,6 +121,7 @@
 texinfo_documents = [
   ('index', 'Kinto', u'Kinto Documentation',
    u'Mozilla Services — Da French Team', 'Kinto',
-   'A storage service for data sync, share with fine-grained permissions.',
+   'A storage service for data storage, synchronization and sharing '
```

Comment:
"A storage service for storage" sounds weird to me. The fact that we have fine grained permissions is IMO too much detailed information. I would rephrase this with "A remote storage service with syncing and sharing abilities"?